bin/hdfcoinc/pycbc_make_hdf_coinc_workflow

```diff
@@ -141,6 +144,17 @@ if len(inj_files) > 0:
     wf.make_sensitivity_plot(workflow, found_inj, 'plots/sensitivity', 
                                             tags=['ALLINJ'])
 
+# save global config file to results directory
+ini_file_path = 'plots/configuration.ini'
+with open(ini_file_path, 'wb') as ini_fh:
+    container.cp.write(ini_fh)
```

Comment:
Tito, 

I'm not sure why you need the following lines. It looks like you've already written the configuration file to the plots directory (which we should rename at some point...). That should be all you need for it to show up in the results page.